assets/javascripts/discourse/templates/components/chat-composer-uploads.hbs

```diff
@@ -0,0 +1,19 @@
+{{#if showUploadsContainer}}
+  <div class="chat-composer-uploads-wrapper">
+    {{#each uploads as |upload|}}
+      {{chat-composer-upload
+        upload=upload
+        isDone=true
+        onCancel=(action "removeUpload" upload)
+      }}
+    {{/each}}
+    {{#each inProgressUploads as |upload|}}
```

Comment:
It's interesting we don't have a "cancel all uploads" button in chat. IDK if we need it or not, just observing. In core this sets `uploadCancelled` on the `composerModel`:

https://github.com/discourse/discourse/blob/a0eddd0518ccd619a0a7661431cbaa7d6a93a98e/app/assets/javascripts/discourse/app/controllers/composer.js#L456-L458